libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java

```diff
@@ -92,6 +92,7 @@ void sendAttachment(Attachment attachment, final CopyOnWriteArraySet<AttachmentL
 
```

Comment:
Thank for changes. Is it possible to add extra logs here as we did for `sendBatch` ?
I mean something like 
`if (isExtraDebuggingNeeded()) { 
     logger.debug(LOG_TAG, String.format(Locale.US, EXTRA_DEBUGGING_LOG, url, visionAttachments.size(), userAgent, metadataList));	
}`